cmake/Modules/FindTPLHPX.cmake

```diff
@@ -0,0 +1,15 @@
+
+FIND_PACKAGE(HPX REQUIRED)
+#as of right now, HPX doesn't export correctly
+#so let's convert it to an interface target
+KOKKOS_CREATE_IMPORTED_TPL(HPX INTERFACE
+  LINK_LIBRARIES ${HPX_LIBRARIES}
+  INCLUDES ${HPX_INCLUDE_DIRS}
+)
+#this is a bit funky since this is a CMake target
+#but HPX doesn't export itself correctly
+KOKKOS_EXPORT_CMAKE_TPL(HPX)
+
+#I would prefer all of this gets replaced with
+#KOKKOS_IMPORT_CMAKE_TPL(HPX)
```

Comment:
Is there somebody we could request this from? Changes to how HPX CMake works? Not a blocking comment, a question

Reply:
I've actually chatted with them and they've been working on it. They have a PR open to do exactly that.